tests/qchem/of_tests/test_convert.py

```diff
@@ -813,3 +813,47 @@ def dummy_ansatz(phis, wires):
     res = dummy_cost(phis)
 
     assert np.abs(res - expected_cost) < tol["atol"]
+
+
+@pytest.mark.parametrize(
+    ("electrons", "orbitals", "singles_ref", "doubles_ref"),
+    [
+        # trivial case
+        (2, 4, [[0, 2], [0, 3], [1, 2], [1, 3]], [[0, 1, 2, 3]]),
+    ],
+)
+def test_excitations(electrons, orbitals, singles_ref, doubles_ref):
+    r"""Test if the _excitations function returns correct single and double excitations."""
+    singles, doubles = qchem.convert._excitations(electrons, orbitals)
+    assert singles == singles_ref
+    assert doubles == doubles_ref
+
+
+@pytest.mark.parametrize(
+    ("electrons", "orbitals", "excitation", "states_ref", "signs_ref"),
+    [
+        # reference data computed with pyscf:
+        # pyscf_addrs, pyscf_signs = tn_addrs_signs(orbitals, electrons, excitation)
```

Comment:
should add the pyscf.fci.cistring here to make it clear where the function lives -- unless this refers to our function, in which case it should be renamed

Reply:
Added.